### PR TITLE
Pin npm version to avoid incompatibility with node version

### PR DESCRIPTION
**Pull request by liam-lloyd**

### CODE REVIEW

File: provisioners/configure.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ envsubst < $TEMPLATES_PATH/etc/permanent/notification-service.env > /etc/permane
 systemctl enable notification.service
 
 echo "Install node global packages"
-npm install npm --global
+npm install npm@8.17.0 --global
```
Comment (slifty) [inline provisioners/configure.sh:117]: This looks good.  I'm curious if we have a sense why we're bothering to install a different version of npm in the first place.

Comment (liam-lloyd) [reply provisioners/configure.sh:117]: I don't have a clear idea of why we're doing that. The version installed by default (6.14.16) is a few major versions behind current, but we're also using a version of node that's several major versions back.

**Review by slifty — APPROVED** [review]
This does look reasonable to me (we may consider removing this line entirely when we upgrade debian / node, but for now it makes more sense to do this than anything else I suppose).

**Review by cecilia-donnelly — APPROVED** [review]
I reprovisioned my local instance from this branch and confirmed that npm is at this version (it was on 8.18 before) and that I can upload to my local copy. Thanks, @liam-lloyd !

**Pull request merged**